packages/components/src/components/accordion/accordion.lite.tsx

```diff
@@ -88,7 +88,7 @@ export default function DBAccordion(props: DBAccordionProps) {
 
 	onUpdate(() => {
 		if (ref && state._initOpenIndexDone) {
-			if (props.initOpenIndex?.length > 0) {
+			if (props?.initOpenIndex && props.initOpenIndex?.length > 0) {
```

Comment:
```suggestion
			if (props?.initOpenIndex?.length > 0) {
```

shouldn't this this be the equivalent to a chaining like this ?

Reply:
Nope. Some typescript configs will mark this as an error

Reply:
> Nope. Some typescript configs will mark this as an error

can we remove that "typescript" ?